views/components/Sidebar/SidebarFooter.test.jsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * @jest-environment jsdom
+ */
+import React from 'react'
+import { MemoryRouter } from 'react-router-dom'
+import { render, waitFor } from '@testing-library/react'
+import SidebarFooter from './SidebarFooter'
+import { createUser } from '../../../test/fixtures'
+
+describe('SidebarFooter', () => {
+  const user = createUser()
+  test('renders SidebarFooter component with user data', async () => {
+    const { getByText, getByAltText } = render(
+      <MemoryRouter>
+        <SidebarFooter user={user} />
+      </MemoryRouter>
+    )
+
+    await waitFor(() => {
+      const userNameElement = getByText(user.display_name)
+      const userAvatarElement = getByAltText('D')
+
+      expect(userNameElement).toBeDefined()
+      expect(userAvatarElement).toBeDefined()
+    })
+  })
+
+  test('renders Edit Profile and Log Out buttons with correct links', async () => {
+    const { getByText } = render(
+      <MemoryRouter>
+        <SidebarFooter user={user} />
+      </MemoryRouter>
+    )
+
+    await waitFor(() => {
+      const editProfileButton = getByText('Edit Profile')
+      const logOutButton = getByText('Log Out')
+
+      expect(editProfileButton).toBeDefined()
+      expect(editProfileButton).toHaveAttribute('href', '/user-account')
+      expect(logOutButton).toBeDefined()
+    })
```

Comment:
I believe the recommended style is to have only 1 expectation in the waitFor

```suggestion
    await waitFor(() => {
      const editProfileButton = getByText('Edit Profile')

      expect(editProfileButton).toHaveAttribute('href', '/user-account')
    })

    const logOutButton = getByText('Log Out')
    expect(logOutButton).toBeDefined()
```

Reply:
[6b310d1](https://github.com/AMP-SCZ/dpdash/pull/458/commits/6b310d18b25129974d8ce470610170164ed5d426)